src/linux/Packaging.Linux/build.sh

```diff
@@ -49,28 +49,10 @@ if [ ! -d "$INSTALL_PREFIX" ]; then
     mkdir -p "$INSTALL_PREFIX"
 fi
 
-# Fall back to host architecture if no explicit runtime is given.
-if test -z "$RUNTIME"; then
-    HOST_ARCH="`dpkg-architecture -q DEB_HOST_ARCH`"
```

Comment:
How about we just move away from using the `dpkg-architecture` command all together? That way we don't need to rely on something that isn't always installed.

`uname -m` should give us what we want.

```shell
if test -z "$RUNTIME"; then
    HOST_ARCH="$(uname -m)"

    case "$HOST_ARCH" in
        x86_64)
        amd64)
            RUNTIME="linux-x64"
            ;;
        arm64)
        aarch64)
            RUNTIME="linux-arm64"
            ;;
        arm)
        armhf)
        aarch32)
            RUNTIME="linux-arm"
            ;;
        *)
            die "Unknown or unsupported host architecture: '$(HOST_ARCH)'"
            ;;
    esac
fi
```

Reply:
nope. that is unfortunately a very common misconception. uname gives the kernel architecture. it knows nothing of the system OS architecture. these commonly do not match as many 32bit distros use 64bit kernels.

Reply:
Not to mention we also cannot detect what libc (glibc/musl) the running OS uses based on the kernel. Using uname is totally flawed.